### PR TITLE
Synology homemode

### PR DESCRIPTION
**Pull request by philklei**
**Description:**
Added description for Synology Home Mode

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12088

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by arsaboo** [comment]
Is this PR about Synology? You have some Tahoma related PRs mixed up. Can you please clean this PR up a little.

**Review by fabaff — CHANGES_REQUESTED** [review]
Can't be merged. The Tahoma docs must be removed.

**Comment by frenck** [comment]
I will rebase the last commit to a fresh branch.
Adding it to my todo list.

**Comment by frenck** [comment]
Rebased the last 2 commits onto `next`. Ready for review now.

File: source/_components/switch.synology_homemode.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: synology_homemode
+    url: http://yousyno:5000
+    username: xxxxxxxx
+    password: xxxxxxxx
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/switch.synology_homemode.markdown:29]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@philklei I've reviewed your PR and left a comment. Could you please take a look? 👍

**Comment by frenck** [comment]
@philklei Little heads up, still waiting for a response.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed as well.

Feel free to re-open as soon as you are ready.

**Pull request closed without merging**